.github/CODEOWNERS

```
@@ -0,0 +1,2 @@
+# Automatically request reviews from the synapse-core team when a pull request comes in.
+*   @matrix-org/synapse-core
```

```suggestion
* @matrix-org/synapse-core
```
nit: I think the extra whitespace may be superfluous